service/http.directory/src/test/java/org/kaazing/gateway/service/http/directory/HttpDirectoryServiceIT.java

```diff
@@ -261,6 +265,33 @@ public void testInvalidUri() throws Exception {
     public void testUriWithParams() throws Exception {
         robot.finish();
     }
+    
+    @Specification("get.file.directory")
+    @Test()
+    public void testGetFileInDirectory() throws Exception {
+    	robot.finish();
+    }
+    
+    @Specification("get.no.file")
+    @Test()
+    public void testGetNoFile() throws Exception {
```

Comment:
Looks like this test already exists, no need to have it twice (name is get non existant file)